bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -23,6 +23,12 @@
 			<description>IP-Address of the Shelly device.</description>
 			<context>network-address</context>
 		</parameter>
+		<parameter name="weakSignal" type="integer" required="false">
+			<label>Weak Signal Threshold</label>
+			<description>An alarm is triggered if a device reports WiFi signal strength below this threshold. Default: -80 dBm</description>
```

Comment:
This feature does imho not make much sense in a binding - it is not at all specific here.
If a user wishes such an alarm, wouldn't it be hundred times better to have the alarm cover ALL devices that report their signal strength and not just the Shelly ones? This hence belongs into a rule and not in a binding.

Reply:
ok, but then it should be a OH core feature. The binding can't monitor the wifi signal for other devices. Same argument: Wha does each user has to add a rule for each device (or one serving it all). Would be a cool feature for OH3 to extend the channel type with a health monitoring.

Reply:
Yes, the signal strength has been defined as a system channel to allow such feature for the core. The new UI should definitely show the signal strength for things and we could also add further functionality for that.

Reply:
ok, what do you think of
- keep the feature in 2.5
- remove it for 3.0 and integrate such monitoring in core

Reply:
Please no. Stuff that is there usually stays around far too long. And if it is even something that is a workaround for a missing feature, it will be copied by others and it'll be all over the place.

Reply:
This is still here...

Reply:
removed even I don't agree